android/app/src/main/java/com/reactnativenavigation/views/RctView.java

```diff
@@ -71,11 +71,10 @@ public void onTemporallyRemovedFromScreen() {
     }
 
     /**
-     * Must be called before view is removed from screen inorder to ensure onDetachedFromScreen is properly
-     * executed and componentWillUnmount is called
+     * Must be called before view is removed from screen inorder to ensure ReactRootView is unmounted.
      */
     public void onRemoveFromScreen() {
-        ReflectionUtils.setField(mReactRootView, "mAttachScheduled", false);
+        ReflectionUtils.invoke(mReactRootView, "unmountReactApplication");
```

Comment:
unmountReactApplication is public, no need to invoke it using reflection.
From what I can tell this method was introduced in RN 0.26. This will break the library for users that use older versions.
I'll look into this now